src/components/__tests__/autocomplete.test.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { ApolloProvider } from '@apollo/client';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { client } from '../../ApolloClient';
+import { setupServer } from '../../mocks/setupServer';
+import Autocomplete from '../autocomplete';
+
+const autoCompleteSetup = () => {
+  render(
+    <ApolloProvider client={client}>
```

Comment:
The way the client is passed to the `ApolloProvider` the `client` is shared between the tests in this file (test suite). Is this expected?

Reply:
yep, intended :)